prow/pod-utils/decorate/podspec.go

```diff
@@ -152,18 +152,19 @@ func LabelsAndAnnotationsForJob(pj prowapi.ProwJob) (map[string]string, map[stri
 }
 
 // ProwJobToPod converts a ProwJob to a Pod that will run the tests.
-func ProwJobToPod(pj prowapi.ProwJob, buildID string) (*coreapi.Pod, error) {
-	return ProwJobToPodLocal(pj, buildID, "")
+func ProwJobToPod(pj prowapi.ProwJob) (*coreapi.Pod, error) {
```

Comment:
Why was this function signature changed? The build ID should always be available when creating a pod for a ProwJob and including it in the function signature makes the requirement explicit.

Reply:
Thank you for the review. Fixed.